src/Overcrowdin/Overcrowdin.nuspec

```diff
@@ -0,0 +1,34 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
There's no need to add a .nuspec file when using the sdk-style .csproj - `msbuild /t:pack` (or the corresponding `dotnet` command) will use the information from the .csproj file to create the nuget file.

Reply:
@ermshiperete The msbuild pack command does not do an adequate job for making a dotnet global tool.

Reply:
`dotnet pack` should work though - see https://github.com/sillsdev/SIL.NuGetCleaner/ which is a similar project